packages/bruno-app/src/components/Modal/index.js

```diff
@@ -65,10 +68,15 @@ const Modal = ({
   closeModalFadeTimeout = 500
 }) => {
   const [isClosing, setIsClosing] = useState(false);
-  const escFunction = (event) => {
-    const escKeyCode = 27;
-    if (event.keyCode === escKeyCode) {
-      closeModal({ type: 'esc' });
+
+  const handleKeydown = ({ keyCode }) => {
+    const map = new Map([
```

Comment:
I think it would be better to use a switch statement here

Reply:
Thanks for the review !
Yeah both approach are good it only depend on performance accessing (in this case the map version is slightly better).
If it's more understandable with the switch statement I change the code !

Reply:
I think the map approach is pretty straightforward. it could be better to put the map outside this function, though

Reply:
@Its-treason do you want me to change this behaviour or not ?

Reply:
Yes, in my opinion using a switch statement is better than map object, because switch is a control structure and Map is for storing data.

Reply:
Ok it took time to do it but I changed everything